rootpy/stl.py

```diff
@@ -156,6 +156,13 @@
     # try to create it. See https://github.com/rootpy/rootpy/issues/328
     mkdir_p(DICTS_PATH)
 
+include_list = os.path.join(userdata.BINARY_PATH, 'include_paths.list')
+log.debug('Using %s to get additional include paths' % include_list)
```

Comment:
use new-style string formatting: ``'Using {0} to get additional include paths'.format(include_list)``